src/Filter/StringFilter.php

```diff
@@ -21,6 +21,8 @@ class StringFilter extends Filter
 {
     public const CHOICES = [
         ContainsOperatorType::TYPE_CONTAINS => 'LIKE',
+        ContainsOperatorType::TYPE_STARTS_WITH => 'LIKE',
+        ContainsOperatorType::TYPE_ENDS_WITH => 'LIKE',
```

Comment:
This constant doesn't exist currently. In order to use it you'll need the SonataAdminBundle PR to be merged and release, then you'll need to update the required version in the composer.json.

Reply:
So this PR will wait for other PR to get merged.

Reply:
I will update it once the other one get merged. I hope it's right way.